### ISSUE TITLE
cli: cdktf cli references manifest from cdktf

### ISSUE THREAD

**Issue by skorfmann** [hashicorp/terraform-cdk#1033]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

### cdktf & Language Versions

- node v14
- cdktf-cli 0.6.1 (globally installed)

### Affected Resource(s)

- cdktf get

### Debug Output

```
< cdktf get
internal/modules/cjs/loader.js:892
  throw err;
  ^

Error: Cannot find module '../../cdktf/lib/manifest'
Require stack:
```

can be workaround by installing cdktf globally as well, but that's not expected

**Comment by ASteinheiser** [comment]
I appreciate the workaround!

**Comment by sushantdhiman** [comment]
_Workaorund:_ you will need to install `cdktf` globally as well, use `npm install --global cdktf`

**Comment by ansgarm** [comment]
I think we should release the fix for this before 0.7, @skorfmann what do you think?
